services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/SubscriptionStatus.jsx

```diff
@@ -14,6 +14,8 @@ import {
 
 import StripeSubscriptionCheckoutSessionButton from '../shared/StripeSubscriptionCheckoutSessionButton';
 
+const CTA_BUTTON_CLASSNAME = "quill-button medium contained primary focus-on-light"
```

Comment:
I like this, we can probably start doing this in a lot of other places in the code base